pkg/kyc/component.go

```diff
@@ -161,25 +211,43 @@ func (c *component) getUser(ctx context.Context, userID string, kcUser kc.UserRe
 	}
 	res.ImportFromKeycloak(ctx, &kcUser, c.logger)
 
+	if !allDetails {
+		// At this point, we shall not provide too many information if consent is required
+		res = apikyc.UserRepresentation{
+			ID:             res.ID,
+			Username:       res.Username,
+			FirstName:      res.FirstName,
+			LastName:       res.LastName,
+			Email:          res.Email,
+			Accreditations: res.Accreditations,
```

Comment:
I would suggest to keep only:
ID, username, firstname, lastname, phone (as you suggested, it might be usefull to display the phone number on which we send it).
Maybe we could just keep only 2 last digits and country indicator of phone number in the result (e.g. +33XXXXXXXX02)
What do you think of it ?

Reply:
I added a method to obfuscate phone numbers in common-service